packages/ott-vis-datasource/src/datasource.ts

```diff
@@ -20,16 +20,32 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
 		this.baseUrl = instanceSettings.jsonData.baseUrl;
 	}
 
+	async request(url: string, params?: string) {
+		const response = getBackendSrv().fetch<SystemState>({
+			url: `${this.baseUrl}${url}${params?.length ? `?${params}` : ""}`,
+		});
+		return lastValueFrom(response);
+	}
```

Comment:
nit: putting this in a function just makes this more complicated than it needs to be.